docs/commands/move.md

```diff
@@ -0,0 +1,33 @@
+# `move` command
+
+Note: The implementation for `copy` and `move` are exactly the same. The only different is that move will remove the source after a successful copy.
```

Comment:
The "implementations", plural. 
The only "difference" is that `move` (<- code fence).